codec-memcache/src/main/java/io/netty/handler/codec/memcache/FullMemcacheMessage.java

```diff
@@ -20,22 +20,4 @@
  * message. So it represent a <i>complete</i> memcache message.
  */
 public interface FullMemcacheMessage extends MemcacheMessage, LastMemcacheContent {
-
-    @Override
-    FullMemcacheMessage copy();
-
-    @Override
-    FullMemcacheMessage retain(int increment);
-
-    @Override
-    FullMemcacheMessage retain();
-
-    @Override
-    FullMemcacheMessage touch();
-
-    @Override
-    FullMemcacheMessage touch(Object hint);
-
-    @Override
-    FullMemcacheMessage duplicate();
```

Comment:
same comment as above. This is needed as otherwise it will not return the correct type and so not allow method chaining correctly.